### ISSUE TITLE
ubuntu/debian missing scyllatop files

### ISSUE THREAD

**Issue by fruch** [scylladb/scylla#5518]
This is Scylla's bug tracker, to be used for reporting bugs only.
If you have a question about Scylla, and not a bug, please ask it in
our mailing-list at scylladb-dev@googlegroups.com or in our slack channel.

- [x] I have read the disclaimer above, and I am reporting a suspected malfunction in Scylla.

*Installation details*
Scylla version (or git commit hash):  3.1.2-0.20191127.9d78d848e
Cluster size: ?
OS (RHEL/CentOS/Ubuntu/AWS AMI): Ubuntu 16.04

Reported on `scylladb-users`
https://groups.google.com/d/msg/scylladb-users/vbArYuMJyhw/W0PFZhbEBQAJ

I have installed binary package of scylla for Ubuntu 16.04, version 3.1.2-0.20191127.9d78d848e
Scyllatop returns error about missing collectd python module:

```bash
$> scyllatop
Traceback (most recent call last):
  File "/usr/lib/scylla/scyllatop/libexec/scyllatop.py", line 9, in <module>
    import collectd
ModuleNotFoundError: No module named 'collectd'
```

@amoskong found that some of the files are missing:

We didn't correctly install all scyllatop scripts [1] to ubuntu16.04 / debian9.
The collect module is this file https://github.com/scylladb/scylla/blob/master/tools/scyllatop/collectd.py

[1] https://github.com/scylladb/scylla/tree/master/tools/scyllatop
```bash
scylla-test@rolling-upgrade-3-2-ubuntu-xenial-db-node-6a03e71f-0-1:~$ scylla --version
3.1.2-0.20191127.9d78d848e
scylla-test@rolling-upgrade-3-2-debian-stretch-db-node-ec4d0433-0-4:~$ find /usr/lib/scylla/scyllatop/
/usr/lib/scylla/scyllatop/
/usr/lib/scylla/scyllatop/libexec
/usr/lib/scylla/scyllatop/libexec/scyllatop.py
/usr/lib/scylla/scyllatop/scyllatop.py
scylla-test@rolling-upgrade-3-2-debian-stretch-db-node-ec4d0433-0-4:~$ 
```

**Comment by syuu1228** [comment]
@fruch @amoskong You are correct, I found that on branch-3.1 .deb build script mistakenly skipping install tools/scyllatop/*.py.
Sent fix on ML: [PATCH scylla branch-3.1 v2] dist/debian: fix missing scyllatop files

**Comment by slivne** [comment]
whats the status with 3.2 / master ?

**Comment by amoskong** [comment]
The bug doesn't exist with Scylla 3.2 on debian 9.

scyllatop works well.

```
scylla-test@amos-debian9-tmp:~$ dpkg -L scylla-server|grep scyllatop
/opt/scylladb/scyllatop
/opt/scylladb/scyllatop/__pycache__
/opt/scylladb/scyllatop/__pycache__/collectd.cpython-37.pyc
/opt/scylladb/scyllatop/__pycache__/defaults.cpython-37.pyc
/opt/scylladb/scyllatop/__pycache__/dumptostdout.cpython-37.pyc
/opt/scylladb/scyllatop/__pycache__/fake.cpython-37.pyc
/opt/scylladb/scyllatop/__pycache__/livedata.cpython-37.pyc
/opt/scylladb/scyllatop/__pycache__/metric.cpython-37.pyc
/opt/scylladb/scyllatop/__pycache__/parseexception.cpython-37.pyc
/opt/scylladb/scyllatop/__pycache__/prometheus.cpython-37.pyc
/opt/scylladb/scyllatop/__pycache__/scyllatop.cpython-37.pyc
/opt/scylladb/scyllatop/__pycache__/userinput.cpython-37.pyc
/opt/scylladb/scyllatop/collectd.py
/opt/scylladb/scyllatop/defaults.py
/opt/scylladb/scyllatop/dumptostdout.py
/opt/scylladb/scyllatop/fake.py
/opt/scylladb/scyllatop/libexec
/opt/scylladb/scyllatop/libexec/scyllatop.py
/opt/scylladb/scyllatop/livedata.py
/opt/scylladb/scyllatop/metric.py
/opt/scylladb/scyllatop/parseexception.py
/opt/scylladb/scyllatop/prometheus.py
/opt/scylladb/scyllatop/scyllatop.py
/opt/scylladb/scyllatop/userinput.py
/opt/scylladb/scyllatop/views
/opt/scylladb/scyllatop/views/__init__.py
/opt/scylladb/scyllatop/views/__pycache__
/opt/scylladb/scyllatop/views/__pycache__/__init__.cpython-37.pyc
/opt/scylladb/scyllatop/views/__pycache__/aggregate.cpython-37.pyc
/opt/scylladb/scyllatop/views/__pycache__/base.cpython-37.pyc
/opt/scylladb/scyllatop/views/__pycache__/groups.cpython-37.pyc
/opt/scylladb/scyllatop/views/__pycache__/helpers.cpython-37.pyc
/opt/scylladb/scyllatop/views/__pycache__/mergeable.cpython-37.pyc
/opt/scylladb/scyllatop/views/__pycache__/simple.cpython-37.pyc
/opt/scylladb/scyllatop/views/__pycache__/stdout.cpython-37.pyc
/opt/scylladb/scyllatop/views/__pycache__/table.cpython-37.pyc
/opt/scylladb/scyllatop/views/aggregate.py
/opt/scylladb/scyllatop/views/base.py
/opt/scylladb/scyllatop/views/groups.py
/opt/scylladb/scyllatop/views/helpers.py
/opt/scylladb/scyllatop/views/mergeable.py
/opt/scylladb/scyllatop/views/simple.py
/opt/scylladb/scyllatop/views/stdout.py
/opt/scylladb/scyllatop/views/table.py
/opt/scylladb/bin/scyllatop
/usr/bin/scyllatop
scylla-test@amos-debian9-tmp:~$ dpkg -L scylla-server|grep scyllatop|grep collect
/opt/scylladb/scyllatop/__pycache__/collectd.cpython-37.pyc
/opt/scylladb/scyllatop/collectd.py
scylla-test@amos-debian9-tmp:~$ 
```

**Comment by syuu1228** [comment]
@slivne as amos reported, the issue not exist on branch-3.2 nor master.
Only on branch-3.1 (verified by checking build script)

**Comment by slivne** [comment]
fixed in 478c35e07a03a5a521d834f33cf4af5e3c0b7188

will be part of the next release

**Comment by slivne** [comment]
no-backport is needed only in 3.1
